bots/make-checkout

```diff
@@ -67,7 +67,10 @@ def main():
             execute("git", "remote", "remove", "test")
         execute("git", "remote", "add", "test", "https://github.com/" + opts.repo)
 
-    execute("git", "fetch", "test" if opts.repo else "origin", opts.ref)
+    # if the pr was updated while the command was in the queue, the revision
+    # won't be tied to the ref anymore, and it won't exist locally, so exit
+    # silently
+    execute("git", "fetch", "test" if opts.repo else "origin", opts.ref, permissive=True)
```

Comment:
My connotation of `permissive` is to ignore errors, not to abort the program with a zero exit code. But I can't think of a better name, as this is a really weird semantics for `execute()`. How about you don't use that at all, and just do `subprocess.call()` instead, and exit here on failure? (You'd need to replicate the verbose logging)

Also, this should in principle apply to the current infra as well, so could be a separate commit.